### PR TITLE
Create new parser ShellTv and Detect new brands: Elekta, Loview, Yuno, YASIN, AMCV, ECON, Erisson, Leben, Novex, Starwind, Vargo, Gooweel and Detect devices for existing brands: Leagoo, CUBOT, Ginzzu, Ulefone, ZTE, Alcatel, Quantum, Ark, BB Mobile

### PR DESCRIPTION
**Pull request by sanchezzzhak**

### CODE REVIEW

File: Parser/Device/HbbTv.php
Comment (sanchezzzhak) [inline Parser/Device/HbbTv.php:61]: @sgiehl I added a new condition to this parser, or was it better to do it in mobiles.yml?

Comment (sgiehl) [reply Parser/Device/HbbTv.php:61]: guess it's fine that way. But we may need to release a new major version after merging to ensure the fork maintainers may have a closer look on the changes

Comment (sanchezzzhak) [reply Parser/Device/HbbTv.php:61]: ok, give me a couple of days (2~) I'll add more tv brands to this PR

File: regexes/device/televisions.yml
Comment (sanchezzzhak) [inline regexes/device/televisions.yml:805]: @sgiehl this is not an exact definition of the model name, from such a user agent we can only get the matrix size and screen type in some cases.
or is it better to leave the model empty string?

Comment (sgiehl) [reply regexes/device/televisions.yml:805]: Isn't something like `LC390TA2A` the model name in this case? Brands like LG actually have similar models.

Comment (sanchezzzhak) [reply regexes/device/televisions.yml:805]: this is not the name of the device model, it is the name of the panel and the motherboard

https://www.panelook.com/LC390TA2A_Panda_39_CELL_overview_26807.html

this applies to the useragent in the following format
<brand> shell <panel code or framework>

* Mozilla/5.0 (Linux; Android 4.4.4; Mystery Shell 536D315LH71_HD) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/73.0.3683.90 Safari/537.36
* Mozilla/5.0 (Linux; Android 4.4.4; JVC Shell 536D315LH71_HD Build/KTU84P) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/33.0.0.0 YaBrowser/19.5.0.155 (lite) Safari/537.36

* Mozilla/5.0 (Linux; Android 4.4.4; Mystery Shell 536D315XH33_HD) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/77.0.3865.116 Safari/537.36
* Mozilla/5.0 (Linux; Android 4.4.4; JVC Shell 536D315XH33_HD Build/KTU84P) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/33.0.0.0 YaBrowser/19.5.0.155 (lite) Safari/537.36

* Mozilla/5.0 (Linux; Android 4.4.4; WildRed Shell V400HJ6_PE1) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/77.0.3865.116 Safari/537.36
* Mozilla/5.0 (Linux; Android 4.4.4; ERISSON_shell_V400HJ6_PE1_C3_40DN4_HD_W18023) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/75.0.3770.143 Safari/537.36

Comment (sgiehl) [reply regexes/device/televisions.yml:805]: Ah ok. I see. Hm... I actually don't have a big preference in this case. Guess we can also leave it empty like for a couple of other TV brands, were the model can't be detected

File: Parser/Device/HbbTv.php
Comment (sgiehl) [inline Parser/Device/HbbTv.php:42]: Wondering if we should rename the class, as it now also handles ShellTV 🤔. Or alternatively we move all the ShellTV detections to a new parser, that only handles the shelltv stuff.

Comment (sanchezzzhak) [reply Parser/Device/HbbTv.php:42]: I think it's better to make a separate TvParser

Comment (sgiehl) [reply Parser/Device/HbbTv.php:42]: You can also add an additional `ShellTvParser`. The advantage of having separate parsers is, that they can be used standalone if someone needs it.

File: Tests/Parser/Device/ShellTvTest.php
Comment (sgiehl) [inline Tests/Parser/Device/ShellTvTest.php:20]:
```suggestion
    public function testIsShellTv(): void
```

File: regexes/device/televisions.yml
Comment (sgiehl) [inline regexes/device/televisions.yml:257]: Can a JVC Shell useragent also include HbbTV? Or is this unneeded?

**Review by sgiehl — CHANGES_REQUESTED** [review]

File: regexes/device/televisions.yml
Comment (sanchezzzhak) [reply regexes/device/televisions.yml:257]: no it can't, it's a mistake

**Review by sgiehl — APPROVED** [review]

**Pull request merged**
